src/main/java/uk/gov/register/presentation/view/LinkHeaderExtractor.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.register.presentation.view;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.stream.Stream;
+
+class LinkHeaderExtractor{
+    private final HttpServletResponse httpServletResponse;
+
+    public LinkHeaderExtractor(HttpServletResponse httpServletResponse) {
+        this.httpServletResponse = httpServletResponse;
+    }
+
+    String extractLink(String linkRel) {
+        String linkHeader = httpServletResponse.getHeader("Link");
+
+        if (StringUtils.isEmpty(linkHeader)) {
+            return null;
+        }
+
+        return Stream.of(linkHeader.split(","))
```

Comment:
put the link url in the `View` object as well as in the response header, as I've suggested before